site/content/docs/install.md

```diff
@@ -1,444 +1,115 @@
-> You can use MariaDB 10.0 or greater as well.  We have tested and validated that
-> the schema and settings work with MariaDB >= 10.0
-> 
-> You can get instructions for installing MariaDB at [MariaDB Repositories](https://downloads.mariadb.org/mariadb/repositories/#mirror=digitalocean-nyc).  Make sure you **select 10.0**
-
-
-```
-sudo apt-get install mysql-server-5.6
-```
-
-* Install will prompt for a mysql root password, use one for the primary mysql user 'root'
-
-> **NOTE**
-> The root password in mysql is the MYSQL root user account.  This can be any password you would 
-> like to use. It is not related to the platform/system root user or any other user.  This password is only the primary "root" password used when using '**mysql -u root -p**'
-
-* After install, mysql should be running
-
-
-### Login to mysql and create the openbmp database and user account
-
-Apply the below to create the database and user that will be used by the openbmp daemon
-
-> **NOTE**
-> The below defaults the openbmp username to use '**openbmp**' as the password.  Change the  
-> identified by '**openbmp'** to something else.
-
-```
-mysql -u root -p
-
-   create database openBMP;
-   create user 'openbmp'@'localhost' identified by 'openbmp';
-   create user 'openbmp'@'%' identified by 'openbmp';
-   grant all on openBMP.* to 'openbmp'@'localhost';
-   grant all on openBMP.* to 'openbmp'@'%';
-```
-
-### MySQL Temporary Table Space
-
-Large queries or queries that involve sorting/counting/... will use a temporary table on disk.   We have found that using a **tmpfs** will improve performance. 
-
-
-#### Create tmpfs (as root)
-
-The below will also configure the tmpfs to be mounted upon restart/boot.
-
-    mkdir -p /var/mysqltmp
-    echo "tmpfs /var/mysqltmp tmpfs defaults,gid=mysql,uid=mysql,size=2400M,mode=0777 0 0" >> /etc/fstab
-    mount /var/mysqltmp
-
-
-### Update the /etc/my.cnf file to enable InnoDB and tune memory
-
-The below **MUST** but adjusted based on your memory available.  Ideally it should be set as high as possible. Below is for a system that has 16G of RAM and 8vCPU.
-
-> #### IMPORTANT
-> You must define **max\_allowed\_packet** to **384M** or greater to support
-> the bulk inserts/updates, otherwise you will get errors that indicate packet is
-> is too large or that the server connection has gone away. 
-
-* sudo vi /etc/mysql/my.cnf
-
-Under **[mysqld]** section
+# add the below to /etc/sysconfig/docker if you have proxies
+export http_proxy="http://proxy.blah.com:80"
+export https_proxy="http://proxy.blah.com:80"
 
+# Start docker
+service docker start 
 ```
-# use the tmpfs mount point
-tmpdir      = /var/mysqltmp
-
-key_buffer_size     = 128M
-
-# This is very IMPORTANT, must be high to handle bulk inserts/updates
-max_allowed_packet  = 384M
-
-net_read_timeout    = 45
-thread_stack        = 192K
-thread_cache_size   = 8
 
-# This value should be roughly 80% of system memory
-innodb_buffer_pool_size = 12G
+### Ubuntu 14.04 (trusty)
```

Comment:
Let's change this to "All Linux Distributions" and then change the below text to direct them to the docker install instructions.  

The reason for this change is because a lot has changed with docker and installing is easier now than it was before. 

You can then remove the whole section on installing for Ubuntu 14.04.